README.md

```diff
@@ -18,16 +18,33 @@ There are three primary goals of this repo:
 
 The recommended way to get started is to browse the [website](https://natsbyexample.com) which provides nicer navigation and presentation for the example code in this repo.
 
-When you want to actually execute the example code, you can clone this repo, download the [`nbe`](https://github.com/ConnectEverything/nats-by-example/releases) CLI and use the `run` command at the root of the repo. For example:
+When you want to actually execute the example code, you need to:
 
+**Pre-requisite requirement:** currently the nbe CLI needs [Docker](https://docs.docker.com/) and [Compose](https://docs.docker.com/compose/) (v2+) to work. It runs a set of containers hosting the CLI client and the NATS server. Other container runtimes would be considered if requested (such as [Podman](https://podman.io/)).
+Install [Docker](https://docs.docker.com/) and [Compose](https://docs.docker.com/compose/) (if you do not have them installed).
+Make sure that `docker` is installed and up, with command:
+```sh
+$ sudo systemctl start docker
+```
```

Comment:
These lines can be removed since they are system specific. Some people may have Docker Desktop installed on macOS or Windows for example, so Docker install instructions are out of scope for this README. Just leave it as Docker and Compose are pre-requisites.